test/fixtures/products.yml

```
@@ -4,14 +4,14 @@ apple:
   photo_url: just a string
   description: red apple 
   stock: 5
-  merchant_id: merchant_one.id
+  merchant_id: 1
   active: true
 banana: 
   title: banana
   price: 4.40
   photo_url: url info
   description: yellow banana
   stock: 5
-  merchant_id: merchant_one.id
+  merchant_id: 1
```

See my comment above for an explanation.

```suggestion
  merchant: merchant_one
```